docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -579,3 +584,18 @@ Make sure you have completed all the steps.
 
 ****
 ////
+
+== Defining metadata for your walkthrough repo
+
+By default, the walkthroughs from a repo are displayed in Solution Explorer with a title corresponding to your repo name. However, you can add a `walkthroughs-config.json` file with the following contents if, for example, you want to title the group `My Solution Patterns`:
```

Comment:
> title corresponding to your repo name

This is written generically enough that it will probably prove to be true. The current thought is that for default we will use something like 'Walkthroughs from <repo-name>". So I think we're good for now, but we may need to change and/or be more specific once the design is finalized.

Reply:
LGTM